src/App/Osu/OsuDifficultyCalculator.cpp

```diff
@@ -942,15 +934,36 @@ double OsuDifficultyCalculator::DiffObject::calculate_difficulty(const Skills::S
 	// RelevantNoteCount @ https://github.com/ppy/osu/blob/master/osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs
 	if (outRelevantNotes)
 	{
-		double maxStrain = (objectStrains.size() < 1 ? 0.0 : *std::max_element(objectStrains.begin(), objectStrains.end()));
-		if (objectStrains.size() < 1 || maxStrain == 0.0 || !osu_stars_xexxar_angles_sliders.getBool())
+		const auto diffObjCompare = [=](const DiffObject& x, const DiffObject& y)
```

Comment:
Inline function declarations including lambdas can have their opening { on the same line (again subjective style)

Reply:
Also a space after the capture [=] before the rest